modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -344,6 +356,36 @@ class Instrument_Manager extends \NDB_Menu_Filter
                 break;
             }
         }
+        // For every value in the `Field` column in the SQL schema, ensure a
+        // matching value exists in the linst file.  This is done to check
+        // whether a question has been deleted from the linst file but is still
+        // present in the back-end.
```

Comment:
The point of checking the schema is to ensure that clicking "Save" doesn't break. There's no harm (and it's common) to not delete old columns so that data is still there if needed.

Reply:
@xlecours can you comment here? I issued this PR as a response to your Redmine ticket so perhaps you can provide more context.